app/connectors/Addresses.scala

```diff
@@ -19,17 +19,16 @@ package connectors
 import javax.inject.Inject
 import models.registration.GroupAccountDetails
 import models.{Address, DetailedAddress}
-import models.{Address, DetailedAddress}
 import play.api.libs.json.{JsDefined, JsNumber, JsValue}
 import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
-import uk.gov.hmrc.play.config.ServicesConfig
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class Addresses @Inject()(config: ServicesConfig, http: HttpClient)(implicit executionContext: ExecutionContext) {
 
-  val url = config.baseUrl("property-linking") + "/property-linking/address"
+  lazy val url = config.baseUrl("property-linking") + "/property-linking/address"
```

Comment:
why is this lazy.

Reply:
i think that's how i found it, made all of them un-lazy now :-D